common/config/rush/command-line.json

```diff
@@ -83,12 +83,20 @@
       "shellCommand": "rush update --variant stable",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "global",
+      "name": "update:all-flavors",
+      "summary": "Update dependencies for all flavors",
+      "description": "Update dependencies for all flavors.",
+      "shellCommand": "rush update && rush update --variant stable",
```

Comment:
it might be a good idea to use `rush update:stable` instead of  `rush update --variant stable` (in case if it the command is updated, we will need to update it in 1 place only)